src/main/java/school/lemon/changerequest/java/vehicles/Boat.java

```diff
@@ -0,0 +1,95 @@
+package school.lemon.changerequest.java.vehicles;
+
+public class Boat extends Vehicle implements IBoat {
+
+    private boolean motorized;
+
+    Boat(int manufacturedYear, String make, String model, boolean motorized) {
+        super(manufacturedYear, make, model);
+        this.motorized = motorized;
+    }
+
+    /**
+     * Is boat motorized
+     *
+     * @return true if motorized, false - otherwise
+     */
+    public boolean isMotorized() {
+        return motorized;
+    }
+
+    /**
+     * Specify whether boat is motorized
+     *
+     * @param motorized whether boat is motorized or not
+     */
+    public void setMotorized(boolean motorized) {
+        this.motorized = motorized;
+    }
+
+
+    /**
+     * @return return string representation in the following formats
+     * for cars: This car is a 1999 Toyota Corrola with 140 hp.
+     * for boats: This boat is a 1980 Bayliner Extreme (with motor).
+     * for airplanes: This airplane is a 1998 ABC Motors Comac that can reach 10000 feet.
+     */
+    public String toString() {
+        String result = String.format("This boat is a %d %s %s",
+                manufacturedYear, manufacturer, model);
+        if (motorized)
+            result += " (with motor).";
+        else
+            result += ".";
```

Comment:
You should never do raw String concatenation, always use `StringBuilder` for dynamic strings. Also, `StringBuilder` in most cases is preferable way to implement `toString` methods, as you never know how would your pattern change and whether new fields would be added.